clang/test/dpct/helper_files_ref/include/device.hpp

```diff
@@ -340,24 +321,41 @@ class device_ext : public sycl::device {
     // Guard the destruct of current_queues to make sure the ref count is safe.
     lock.lock();
   }
+  sycl::property_list get_property_list() const {
```

Comment:
I see that it is a public method of device_ext class. I suggest to make it private. I did not see any usage of it in the generated code.
Also, I suggest to rename to something like: "get_default_property_list_for_queue" to make the name more specific to what the method does actually.

Reply:
OKay